### ISSUE TITLE
Glazed Terracotta is not directional

### ISSUE THREAD

**Issue by valchiara** [Hekera/ThermalDyenamics#13]
Glazed Terracotta from Dyenamics cannot be rotated like Vanilla Glazed Terracotta

**Comment by JaisDK** [comment]
Do you have a picture of that?

**Comment by valchiara** [comment]
Attempting to rotate the block using the Supplementaries wrench:
![2024-07-24_12 39 14](https://github.com/user-attachments/assets/8641a3f8-4baf-4e38-98d2-92c45cda36b6)

**Comment by JaisDK** [comment]
if you place it from a different angle does it not place it rotated?

**Comment by valchiara** [comment]
Correct, it places facing the same direction, regardless of placement angle

**Comment by JaisDK** [comment]
which version are you on?

**Comment by valchiara** [comment]
MC 1.20.1
Forge 47.3.0
Dyenamics 1.20.1-3.2.0

**Comment by Lethosos** [comment]
Poking at the repository myself, I can see there's no defined rotations in the blockstate for glazed terracotta.

**Comment by veccyboi** [comment]
i believe this can be fixed with just a resource pack that adds the rotations back in.
